src/libs/ReportActionsUtils.js

```diff
@@ -76,7 +97,55 @@ function isConsecutiveActionMadeByPreviousActor(reportActions, actionIndex) {
     return currentAction.action.actorEmail === previousAction.action.actorEmail;
 }
 
+/**
+ * Get the message text for the last action that was not deleted
+ * @param {Number} reportID
+ * @param {Object} [actionsToMerge]
+ * @return {String}
+ */
+function getLastVisibleMessageText(reportID, actionsToMerge = {}) {
+    const parser = new ExpensiMark();
+    const actions = _.toArray(lodashMerge({}, allReportActions[reportID], actionsToMerge));
+    const sortedActions = _.sortBy(actions, 'sequenceNumber');
+    const lastMessageIndex = _.findLastIndex(sortedActions, action => (
+        !isDeletedAction(action)
+    ));
+    const htmlText = lodashGet(actions, [lastMessageIndex, 'message', 0, 'html'], '');
```

Comment:
NAB, but I think this also works and is a little cleaner (not tested):
```suggestion
    const htmlText = lodashGet(actions, `[${lastMessageIndex}].message[0].html`, '');
```
From the docs:
![image](https://user-images.githubusercontent.com/1228807/189980066-1c83e97d-c4c5-4ee6-b110-6a2fc10c22e7.png)

Reply:
Ah, if that's your preference, then I think it's fine. Both is acceptable!